src/views/WorkspaceDetail.vue

```diff
@@ -228,7 +228,8 @@ export default Vue.extend({
           if (err.response.status === 409) {
             this.requestError = 'A workspace by that name already exists';
           } else {
-            this.requestError = err.response.statusText;
+            console.log(err.response);
```

Comment:
Could you get rid of this `console.log()` statement?

Reply:
See 8d68860. I've also opened #206 to stop us (mostly me, lol) from committing code that doesn't pass the production linter